copy/copy.go

```diff
@@ -546,6 +551,25 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 		return nil, "", "", errors.Wrapf(err, "Error initializing image from source %s", transports.ImageName(c.rawSource.Reference()))
 	}
 
+	options.DestinationManifestCheck = true // TODO: Forced enablement for initial PR testing
+	if options.DestinationManifestCheck && options.SignBy == "" {
+		destImageSource, err := c.dest.Reference().NewImageSource(ctx, options.DestinationCtx)
+		if err == nil {
+			retManifest, retManifestType, err = destImageSource.GetManifest(ctx, nil)
+			if err == nil {
+				sourceManifest, _, err := src.Manifest(ctx)
+				if err != nil {
+					return nil, "", "", errors.Wrapf(err, "Error reading manifest from image")
+				}
+
+				if bytes.Equal(sourceManifest, retManifest) {
+					c.Printf("Destination image manifest compares equal\n")
```

Comment:
Maybe
```suggestion
					c.Printf("Destination already contains the expected manifest, not copying the image again\n")
```
or something a bit succinct — but the goal is that the reader understands why no layer/manifest copies show up in the debug output.

Reply:
… a bit _more_ succinct, I mean.

Reply:
`"Destination copy skipped as expected image already present"` ?

Reply:
Probably `s/Destination copy/Copy/`, and maybe even `s| as|,|`; I’m not a native speaker and I’m not sure how much the use of “as” to mean “because” is common.